ego4d/internal/human_pose/camera.py

```diff
@@ -58,14 +58,53 @@ def _create_exo_camera(exo_cam_desc):
         height=exo_cam_desc["image_height"],
         params=params,
     )
-
     return camera
 
 
+# ### Added by jinxu
+# def _create_aria_camera():
```

Comment:
nit: do you need this? Please delete the commented code. If you do need them, leave the functions uncommented but maybe add "_example" to their name or something along those lines.